_pkgdown.yml

```
@@ -1,4 +1,4 @@
-url: https://r4ds.github.io/shinysocial/
+url: https://shinyworks.github.io/shinysocial/
 template:
   bootstrap: 5
```

```suggestion
  bootstrap: 5
  light-switch: true
```
Add something to trigger action re-runs.